pkg/packages.builds

```diff
@@ -3,7 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)\projects\*\*.builds" />
+  	<ProjectExclusions Include="$(MSBuildThisFileDirectory)\projects\*\Microsoft.NETCore.UniversalWindowsPlatform.builds" Condition="'$(OSEnvironment)' != 'Windows_NT'" />
```

Comment:
I think this is fine to unblock the build.  I do think we should take a closer look at OfficialBuildRID mechanism and its use by both projects and builds.  Perhaps it should just create items and we should leave it to the proj/builds to decide how to use those.  That stuff has grown organically and we haven't looked at how to make it generically usefull holistically yet.